docs/tutorials/quick_start_rake_tasks.md

```diff
@@ -0,0 +1,205 @@
+
+
+## Tasks
+
+### gen_hosts
+
+To run:
+
+    rake beaker_quickstart:gen_hosts[provisioner]
+
+This task takes an optional parameter (provisioner), if you leave it empty it will default to 'vmpooler'. You can pass in either 'vmpooler' 
+or 'vagrant'.
+
+The gen_hosts task will create a file 'default_provisioner_hosts.yaml' in acceptance/config.
+
+If the file already exists, it will not be overwritten. This will allow you to play around with the config yourself, either by manually 
+editing the file or by using beaker-hostgenerator to generate a new hosts config.
+
+Vmpooler file (Ubuntu 14 master and agent):
+
+    ---
+    HOSTS:
+      redhat7-64-1:
+        pe_dir: 
+        pe_ver: 
+        pe_upgrade_dir: 
+        pe_upgrade_ver: 
+        hypervisor: vmpooler
+        platform: el-7-x86_64
+        template: redhat-7-x86_64
+        roles:
+        - agent
+        - master
+        - database
+        - dashboard
+        - classifier
+        - default
+      redhat7-64-2:
+        pe_dir: 
+        pe_ver: 
+        pe_upgrade_dir: 
+        pe_upgrade_ver: 
+        hypervisor: vmpooler
+        platform: el-7-x86_64
+        template: redhat-7-x86_64
+        roles:
+        - agent
+        - frictionless
+    CONFIG:
+      nfs_server: none
+      consoleport: 443
+      pooling_api: http://vmpooler.delivery.puppetlabs.net/
+    
+
+
+Vagrant file (Ubuntu 14 master and agent):
+
+    ---
+    HOSTS:
+      ubuntu1404-64-1:
+        pe_dir: 
+        pe_ver: 
+        pe_upgrade_dir: 
+        pe_upgrade_ver: 
+        platform: ubuntu-14.04-amd64
+        hypervisor: vagrant
+        roles:
+        - agent
+        - master
+        - database
+        - dashboard
+        - classifier
+        - default
+      ubuntu1404-64-2:
+        pe_dir: 
+        pe_ver: 
+        pe_upgrade_dir: 
+        pe_upgrade_ver: 
+        platform: ubuntu-14.04-amd64
+        hypervisor: vagrant
+        roles:
+        - agent
+        - frictionless
+    CONFIG:
+      nfs_server: none
+      consoleport: 443
+      box_url: https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm
+      box: puppetlabs/ubuntu-14.04-64-nocm
+
+
+For more info on host generation and what these configs represent see:
+
+* [Creating A Test Environment](creating_a_test_environment.md)
+
+
+### gen_pre_suite
+
+To run:
+
+    rake beaker_quickstart:gen_pre_suite
+    
+This task will generate a default Beaker pre-suite file.
+
+The gen_pre-suite task will create a file 'default_pre_suite.rb' in acceptance/setup.
+
+If the file already exists, it will not be overwritten. This will allow you to edit the pre_suite file yourself if required.
+
+pre_suite file:
+
+    install_puppet
+  
+See the other tutorial docs in this directory for more information on pre-suites:
+
+* [Test Suites](test_suites.md)

```

Comment:
If we're not going to have more than one reference in this list, perhaps it would be better to just have the sentence be:

    See the [Test Suites doc](test_suites.md) in this directory for more information on pre-suites.